internal/db/dump/dump.go

```diff
@@ -100,9 +100,6 @@ func dumpData(ctx context.Context, config pgconn.Config, schema, excludeTable []
 		// "storage",
 		// "supabase_functions",
 		"supabase_migrations",
-		"_analytics",
-		"_realtime",
-		"_supavisor",
```

Comment:
We have to keep this for a few versions because users could be using new cli with and old backup db.